shacl12-core/index.html

```diff
@@ -4298,6 +4275,141 @@ <h4>sh:in</h4>
 				</section>
 			</section>
 		</section>
+				
+		<section id="nonValidation" class="informative">
+			<h2>Non-Validating Property Shape Characteristics</h2>
+			<p>
+				While the previous sections introduced properties that represent validation conditions,
+				this section covers properties that are ignored by SHACL processors.
+				The use of these so-called <dfn data-lt="non-validating property">non-validating properties</dfn> is entirely optional and not subject to formal interpretation contracts.
+				They MAY be used for purposes such as form building or predictable printing of RDF files.
+			</p>
+			<section id="name">
+				<h3>sh:name and sh:description</h3>
+				<p>
+					Property shapes may have one or more <a>values</a> for <code>sh:name</code> to provide human-readable labels for the property in the target where it appears.
+					If present, tools SHOULD prefer those locally specified labels over globally specified labels at the <code>rdf:Property</code> itself.
+					For example, if a form displays a node that is in the target of a given property shape with an <code>sh:name</code>, then the tool SHOULD use the provided name.
+					Similarly, property shape may have values for <code id="description">sh:description</code> to provide descriptions of the property in the given context.
+					Both <code>sh:name</code> and <code>sh:description</code> may have multiple <a>values</a>, but should only have one <a>value</a> per language tag.
+				</p>
+			</section>
+			<section id="order">
+				<h3>sh:order</h3>
+				<p>
+					Property shapes may have one <a>value</a> for the property <code>sh:order</code> to indicate the relative order of the property shape for purposes such as form building.
+					The values of <code>sh:order</code> are decimals.
+					<code>sh:order</code> is not used for validation purposes and may be used with any type of subjects.
+					If present at property shapes, the recommended use of <code>sh:order</code> is to sort the property shapes in an ascending order, for example so that
+					properties with smaller order are placed above (or to the left) of properties with larger order.
+				</p>
+			</section>
+			<section id="group">
+				<h3>sh:group</h3>
+				<p>
+					Property shapes may link to an <a>SHACL instance</a> of the class <code>sh:PropertyGroup</code> using the property <code>sh:group</code> to indicate that
+					the shape belongs to a group of related property shapes.
+					Each group may have additional triples that serve application purposes, such as an <code>rdfs:label</code> for form building.
+					Groups may also have an <code>sh:order</code> property to indicate the relative ordering of groups within the same form.
```

Comment:
"an `rdfs:label`" because it reads "an arr-dee-eff-ess". But does `sh:group` read "an ess-aych-group" or "a shh-group"?
```suggestion
					Property shapes may link to a <a>SHACL instance</a> of the class <code>sh:PropertyGroup</code>
					using the property <code>sh:group</code> to indicate that
					the shape belongs to a group of related property shapes.
					Each group may have additional triples that serve application purposes,
					such as an <code>rdfs:label</code> for form building.
					Groups may also have an <code>sh:order</code> property to indicate
					the relative ordering of groups within the same form.
```